### PR TITLE
Multiple minor fixes

### PR DESCRIPTION
**Pull request by AlanSoto31**
## Description
- Open social network links on a new tab.
- Update bottom navbar icons to match the design.
- Dynamic page titles.
- Favicon.

The purpose of the page titles is below, the default is `Rails World 2024`

About page -> `About Telos Labs`
Landing page -> `Coming soon`
Notifications index page -> `Notifications`
Notifications settings page -> `Notifications settings`
Sessions index page -> `Agenda`
Session show page -> `Session`
Profile show page -> `Profile`
Profile edit page -> `Edit Profile`
Schedule index page -> `My schedule`
Speaker show page -> `Speaker`

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [x] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [x] CI pipeline is passing
- [x] My code follows the conventions of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom
image

image

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="h-full">
   <head>
-    <title>RailsWorld</title>
+    <title><%= content_for?(:page_title) ? yield(:page_title) : "RailsWorld" %></title>
```
Comment (LuigiR0jas) [inline app/views/layouts/application.html.erb:4]:
```suggestion
    <title><%= content_for?(:page_title) ? yield(:page_title) : "Rails World 2024" %></title>
```

**Review by LuigiR0jas — COMMENTED** [review]
Looks good. Just a couple of comments:
1. Can you also adjust the bottom navbar icon spacing so that it matches the design?
Screenshot 2024-08-20 at 7 45 24 p m
Screenshot 2024-08-20 at 7 45 18 p m
2. I think it'll be better to append "- Rails World 2024" to all titles

File: app/views/abouts/show.html.erb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+<%= page_title "About Telos Labs" %>
```
Comment (LuigiR0jas) [inline app/views/abouts/show.html.erb:1]:
```suggestion
<%= page_title "About" %>
```

File: app/views/profiles/show.html.erb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<%= page_title "Profile" %>
```
Comment (LuigiR0jas) [inline app/views/profiles/show.html.erb:1]: When the profile is from the current user, I think this title is ok. But if it's a public profile from somebody else, I think the title should contain the person's name i.e "John Doe's Profile"

File: app/views/sessions/show.html.erb [hunk 4]
```diff
@@ -1,3 +1,5 @@
+<%= page_title "Session" %>
```
Comment (LuigiR0jas) [inline app/views/sessions/show.html.erb:1]: This should be the session's name i.e "Opening Keynote"

File: app/views/speakers/show.html.erb [hunk 5]
```diff
@@ -1,3 +1,5 @@
+<%= page_title "Speaker" %>
```
Comment (LuigiR0jas) [inline app/views/speakers/show.html.erb:1]: This should be the speaker's name i.e "Tobias Lutke"

File: app/views/profiles/show.html.erb (hunk 3 above)
Comment (AlanSoto31) [reply app/views/profiles/show.html.erb]: @LuigiR0jas good eye, I set the page title to the user's name regardless of whether the profile belongs to the current user or not, I think it's more consistent but I'm open to implementing your suggestion.

**Pull request merged**